cdk-web.webpack.config.js

```diff
@@ -65,13 +63,20 @@ module.exports = {
       ["os"]: require.resolve("./webpack/modules/os"),
       ["promptly"]: require.resolve("./webpack/modules/empty"),
       ["proxy-agent"]: require.resolve("./webpack/modules/empty"),
+      [$("node_modules/aws-cdk-lib/aws-lambda-nodejs/lib/index.js")]: $(
+        "webpack/modules/aws-cdk-lib/aws-lambda-nodejs/lib/index.js"
+      ),
+      [$("node_modules/aws-cdk-lib/core/lib/asset-staging.js")]: $(
+        "webpack/modules/aws-cdk-lib/core/lib/asset-staging.js"
+      ),
       [$("node_modules/aws-cdk-lib/core/lib/stage.js")]: $("webpack/modules/aws-cdk-lib/core/lib/stage.js"),
       [$("node_modules/aws-cdk/lib/util/directories.js")]: $("webpack/modules/aws-cdk/lib/util/directories.js"),
       [$("node_modules/console-browserify/index.js")]: $("webpack/modules/console-browserify/index.js"),
     },
   },
   plugins: [
     ...(common.__SERVER ? [new plugins.WebpackMildCompile()] : []),
+    plugins.PreBuildPlugin,
```

Comment:
for the sake of staying consistent please subclass this plugin and `new` it here to stay coherent with what's below it.

Reply:
So the prebuild plugin doesn't allow you to create a new instance of it this way without doing all the config here. I was going to subclass it, overload the constructor, pass in defaults, then read them in during the create method... but that felt like way too much work. Anyway, going to remove it based on your other ideas.